nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/grok/GrokReader.java

```diff
@@ -74,6 +74,8 @@
         "The line of text that does not match the Grok Expression will be appended to the last field of the prior message.");
     static final AllowableValue SKIP_LINE = new AllowableValue("skip-line", "Skip Line",
         "The line of text that does not match the Grok Expression will be skipped.");
+    static final AllowableValue ROUTE_TO_FAILURE = new AllowableValue("route-to-failure", "Route to failure",
```

Comment:
Since it is up to the processor using the GrokReader to determine if something will be routed to failure, should we call this "Failure" or "Produce Error" or something indicating that reader is going to throw an exception?